Recipes.txt

```diff
@@ -0,0 +1 @@
+ # Signature Latte Recipe
```

Comment:
**Review of Joe Git Cafe Project**

- Readability

    For improving Readability, you can use bullet points instead of writing in paragraph format in #1.